org.buhocms.BuhoCMS.yml

```diff
@@ -47,15 +47,14 @@ modules:
       - ./configure --prefix=/${FLATPAK_DEST}
       - make all
       - make install
+      - make clean
```

Comment:
this will do nothing either. `make clean` clear the build directory, but it NEVER part of the package. It's totally pointless